### PR TITLE
Added France

### PR DESCRIPTION
**Pull request by llongour**
Hello 
This PR adds France to the list of supported regions.

Many thanks for this great tool!

### CODE REVIEW

**Comment by jbelien** [comment]
Thanks!
It's deployed : https://welcome.osm.be/france/list 🚀 

Data extraction is currently failing ; I'll check in the next few days (if it's not fixed).

File: translations/messages+intl-icu.zh_TW.xlf [hunk 1]
```diff
@@ -53,6 +53,10 @@
         <source>Democratic Republic of the Congo</source>
         <target>剛果民主共和國</target>
       </trans-unit>
+      <trans-unit id="ehyk73U" resname="France">
+        <source>France</source>
+        <target>法国</target>
```
Comment (LaoshuBaby) [inline translations/messages+intl-icu.zh_TW.xlf:58]: In Traditional Chinese (zh_TW) they use `國` instead of `国`

Comment (jbelien) [reply translations/messages+intl-icu.zh_TW.xlf:58]: Oh, sorry about this!
Could you submit a PR to update the correct translation of "France" in Traditional Chinese (zh_TW) ? Thanks a lot.

Comment (LaoshuBaby) [reply translations/messages+intl-icu.zh_TW.xlf:58]: > Could you submit a PR

#396

File: templates/messages/france/fr/default.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Voici quelques astuces pour bien démarrer :
+
+* Même si nos outils d'édition essayent de rendre facile la cartographie des choses, notre [wiki](https://wiki.openstreetmap.org/wiki/) contient bien plus de détails, vous pourrez y trouver beaucoup d'informations sur la manière de cartographier.  
+Si vous voulez savoir comment "taguer" quelque chose, la solution la plus simple est probablement de chercher avec votre moteur de recherche favori comme ceci : `site:wiki.openstreetmap.org [terme à chercher]` ;
+* Quand vous enregistrez vos modifications, ajoutez un commentaire court mais explicite sur ce que vous avez changé. ([Plus d'informations ici](https://wiki.openstreetmap.org/wiki/Good_changeset_comments)) ;
```
Comment (gendy54) [inline templates/messages/france/fr/default.md:18]: https://wiki.openstreetmap.org/wiki/FR:Bons_commentaires_de_groupe_de_modifications

Comment (jbelien) [reply templates/messages/france/fr/default.md:18]: Hello @gendy54 , could you submit a PR to update the template ? Thanks!

**Comment by jbelien** [comment]
Hello @l-longour, the extent of France (Metropolitan France + Overseas) seems to be too big for OSMCha API.

Could you split it in 2 separated regions ?
Thanks!

**Comment by LaoshuBaby** [comment]
I make a experimental branch in our repository : https://github.com/OSMChina/osm-welcome-tool/tree/Help-France-community-fix-2-problem-mentioned

But I don't know France community's reaction, I also afraid that those maybe unproper. 

(**It looks like impolite to edit other countries' region file. So I don't want to make a initiative PR** (If needed maintainer can make PR from downstream repository) and let this only a experimental branch)

**Comment by llongour** [comment]
Hello, thank you very much for your help, I only just noticed your new posts, I just looked at your repository @LaoshuBaby , I didn't see if the result reflected on the frontend would be a single element "France" or as many elements as the subdivision. If it is well unified on the frontend no problem, if not I will switch to metropolitan France and later add the overseas regions.

**Comment by LaoshuBaby** [comment]
> Hello, thank you very much for your help, I only just noticed your new posts, I just looked at your repository @LaoshuBaby , I didn't see if the result reflected on the frontend would be a single element "France" or as many elements as the subdivision. If it is well unified on the frontend no problem, if not I will switch to metropolitan France and later add the overseas regions.

@l-longour Thank you for reply me!

At that time I separated it to 4 regional geojson file:

```
assets/regions/france-atlanticocean.geojson
assets/regions/france-indianocean.geojson
assets/regions/france-metropolitan.geojson
assets/regions/france-pacificocean.geojson
```

I'm not a developer of welcome tool, but other community's mapper, so I don't know whether this separate is correct. (But at least you can choose the metropolitan file use). If current contribute way of welcome tool haven't change, according to https://github.com/osmbe/osm-welcome-tool/blob/2.x/CONTRIBUTING.md , one region , one `osmcha.id`.

**Pull request merged**